backend/api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("reports")]
+public class ReportController : ControllerBase
+{
+    private readonly ILogger<ReportController> _logger;
+    private readonly ReportService _reportService;
+
+    public ReportController(ILogger<ReportController> logger, ReportService reportService)
+    {
+        _logger = logger;
+        _reportService = reportService;
+    }
+
+    /// <summary>
+    /// List all reports on the asset.
+    /// </summary>
+    /// <remarks>
+    /// Responses are paginated.
+    ///
+    /// Example query:
+    ///
+    ///     /reports?
+    ///
```

Comment:
```suggestion 
```

This PR removes this remark for the other controllers, and it should therefore not be added for the report controller.

Reply:
Agreed, fixed